### PR TITLE
returning the centroid of each cell in the sample dict

### PR DESCRIPTION
**Pull request by estherrolf**
Modifies the sample to additionally return the centroid location for each grid cell in the original label data csvs.

### CODE REVIEW

File: torchgeo/datasets/usavars.py [hunk 1]
```diff
@@ -165,6 +165,9 @@ def __getitem__(self, index: int) -> Dict[str, Tensor]:
                 [self.label_dfs[lab].loc[id_][lab] for lab in self.labels]
             ),
             "image": self._load_image(os.path.join(self.root, "uar", tif_file)),
+            "centroid_loc": Tensor(
+                self.label_dfs[self.labels[0]].loc[id_][["lon", "lat"]]
```
Comment (adamjstewart) [inline torchgeo/datasets/usavars.py:169]: I feel like lat/lon is more common than lon/lat. Should we call the key centroid_log or centroid?

Comment (calebrob6) [reply torchgeo/datasets/usavars.py:169]: fiona returns lon/lats (I think this is standard in geospatial libraries)
E.g.

image

When we write coordinates we usually put latitude first though, e.g. 47.661435 N, -122.116502 E.

**Review by calebrob6 — APPROVED** [review]

File: torchgeo/datasets/usavars.py (hunk 1 above)
Comment (estherrolf) [reply torchgeo/datasets/usavars.py:169]: I'll just return lat and lon separately so that it's as clear as possible - thoughts/objections?

Comment (adamjstewart) [reply torchgeo/datasets/usavars.py:169]: That's fine too. The current solution is fine as long as we document it.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**